.github/workflows/ci.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.58.1
+          toolchain: 1.67.1
```

Comment:
The MSRV in README is 1.41.0, we should set up matrix to test all minor versions since 1.41.0, or update the MSRV in README